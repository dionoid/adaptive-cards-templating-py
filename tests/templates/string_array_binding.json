{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "TextBlock",
            "size": "large",
            "weight": "bolder",
            "text": "Names"
        },
        {
            "type": "Container",
            "items": [
                {
                    "type": "TextBlock",
                    "$data": [
                        "Matt", 
                        "David", 
                        "Thomas"
                    ],
                    "text": "${$data}"
                }
            ]
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.3"
}