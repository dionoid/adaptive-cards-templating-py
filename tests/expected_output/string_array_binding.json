{
  "type": "AdaptiveCard",
  "body": [
    {
      "type": "TextBlock",
      "size": "large",
      "weight": "bolder",
      "text": "Names"
    },
    {
      "type": "Container",
      "items": [
        {
          "type": "TextBlock",
          "text": "Matt"
        },
        {
          "type": "TextBlock",
          "text": "David"
        },
        {
          "type": "TextBlock",
          "text": "Thomas"
        }
      ]
    }
  ],
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.3"
}